### PR TITLE
Added Daml Shell intro and links to/from PQS

### PR DESCRIPTION
**Pull request by hrischuk-da**

### CODE REVIEW

File: docs/2.9.0/docs/query/pqs-user-guide.rst [hunk 1]
```diff
@@ -4,13 +4,6 @@
 Participant Query Store User Guide
 ##################################
 
-Docker image
-************
-
-To get the Participant Query Store (PQS) Docker image, run the following command::
-
-    docker pull digitalasset-docker.jfrog.io/participant-query-store:0.1.0
-
```
Comment (garyverhaegen-da) [inline docs/2.9.0/docs/query/pqs-user-guide.rst:13]: This seems unrelated.

Comment (hrischuk-da) [reply docs/2.9.0/docs/query/pqs-user-guide.rst:13]: Someone put it there but had it as the first item in the page. I moved it after the PQS intro so that a user could at least understand why they might want to get the Docker image.

Comment (garyverhaegen-da) [reply docs/2.9.0/docs/query/pqs-user-guide.rst:13]: That someone was me. I put it there because it's a recurring question from people who already know what PQS and know they want it.

Comment (hrischuk-da) [reply docs/2.9.0/docs/query/pqs-user-guide.rst:13]: Yes.  I just moved it below the introduction.  Is that ok?

**Review by marciw-da — COMMENTED** [review]
Thanks for adding the intro! Made some suggestions for clarity and style.

File: docs/2.9.0/docs/query/pqs-user-guide.rst [hunk 2]
```diff
@@ -32,8 +25,18 @@ The PQS is intended for high throughput and complex queries, for which the Canto
 -  Application operators to take a full snapshot of the ledger (from the start of the ledger to the current latest transaction). Alternatively, they can take a partial snapshot of the ledger (between specific `offsets <https://docs.daml.com/app-dev/grpc/proto-docs.html#ledgeroffset>`__).
 -  Report writers to extract historical data and then stream indefinitely (either from the start of the ledger or from a specific offset).
 
+
+:ref:`daml-shell-header` is a command line tool that leverages PQS to inspect a Daml ledger.
+
 There are many other uses.
```
[the remark this answers is not in the document]
Comment (marciw-da) [reply docs/2.9.0/docs/query/pqs-user-guide.rst:31]: (by "this sentence" I mean the line that says "There are many other uses.")

File: docs/2.9.0/docs/tools/daml-shell/index.rst [hunk 3]
```diff
@@ -3,8 +3,30 @@
...
+help via the help command; each command having its own detailed help support.  
+
+Some of the actions that Daml Shell supports are:
+
+* Find a specific contract and display it. For example, if you have a contract-id, Daml Shell allows you to inspect the corresponding contract using the contract command.
+* Find all events related to a transaction ID.  A transaction is displayed as a series of ledger events (creates, archives, and exercises). 
+* Auto-completion for identifiers such as contract-ids, fully qualified names, and package IDs .
+* List active, inactive, or all contracts for a template using a template FQN as an argument.
+* Apply query and filter facilities to commands to manage the output.
+* Use the diff command to highlight the delta between two contract-ids for the same template.
```
[the remark this answers is not in the document]
Comment (marciw-da) [reply docs/2.9.0/docs/tools/daml-shell/index.rst]: > The `diff` command isn't in the list in the Commands section -- should it be added?

Or do you mean the `compare-contracts` command rather than `diff`?

File: docs/2.9.0/docs/query/pqs-user-guide.rst (hunk 1 above)
Comment (hrischuk-da) [reply docs/2.9.0/docs/query/pqs-user-guide.rst:13]: With the mode the Docker information is close enough to the top to be seen.  I do think the introduction is more important than the Docker information because the first question a potential user has is: "Why am I interested in this?"  The second question is "How can I get it?"

File: docs/2.9.0/docs/query/pqs-user-guide.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (hrischuk-da) [reply docs/2.9.0/docs/query/pqs-user-guide.rst:27]: Good catch.  I moved it below the bulleted list.

File: docs/2.9.0/docs/tools/daml-shell/index.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (hrischuk-da) [reply docs/2.9.0/docs/tools/daml-shell/index.rst]: I changed 'diff' to 'compare-contracts'.  I thought I did that before.  Thanks for catching that.

**Review by marciw-da — APPROVED** [review]

File: docs/2.9.0/docs/query/pqs-user-guide.rst (hunk 1 above)
Comment (garyverhaegen-da) [reply docs/2.9.0/docs/query/pqs-user-guide.rst:13]: There's definitely a tradeoff, but consider that the scenario you describe only applies to each user at most once. They may already know they need it for any number of reasons, among which: they've already read the page in the past, their team is using it and they're just joining the project, or their DA contact told them this is the tool they need for the problem they have.
